### PR TITLE
- soften error handling if setting locale fails due to broken system setup (bsc#1085832)

### PR DESCRIPTION
**Pull request by aschnell**
For https://trello.com/c/RYVaFyPX/351-2-opensuse-p2-1085832-snapper-does-not-work-without-locale-installed.

### CODE REVIEW

File: client/snapper.cc [hunk 1]
```diff
@@ -1522,7 +1523,11 @@ main(int argc, char** argv)
     catch (const runtime_error& e)
     {
 	cerr << "Failed to set locale. Fix your system." << endl;
-	exit(EXIT_FAILURE);
+    }
+
+    if (strcmp(nl_langinfo(CODESET), "UTF-8") != 0)
+    {
+	cerr << "Running in non UTF-8 locale. Setup is unsupported." << endl;
```
Comment (lslezak) [inline client/snapper.cc]: Q: The STDERR messages are not translated in snapper? E.g. the `--help` output few lines above is translated, but that goes to STDOUT...

Comment (lslezak) [reply client/snapper.cc]: Actually this STDERR https://github.com/openSUSE/snapper/pull/403/files#diff-b5c274f368d2fab68ee1bc4f9206df4eR1490 is translated, so I guess these two should be as well...

Comment (aschnell) [reply client/snapper.cc]: AFAIS the first will never be translated since setting the locale failed. Anyway, marked to be translated.

**Comment by jsrain** [comment]
Not sure why the .pot-file is now in git (as it can be generated), but LGTM (assuming pending checks succeed)

**Comment by aschnell** [comment]
Weblate needs the pot file. But still Weblate cannot merge the po files. Sad but true.

**Pull request merged**